README.md

```diff
@@ -109,14 +136,13 @@ it('should reject because of x', () => {
 
 #### Assertions and testing APIs
 
-`assert`, `describe`, `it`, `beforeEach` and `afterEach` are
-available in tests by default - you don't need to import
-anything for those to work.
+`assert`, `describe`, `it`, `beforeEach`, and `afterEach` are
```

Comment:
I'm not worried either way to be honest. I've never used them myself and I'm not likely to, but I'm fine if this is left as is.